config/prow/external-secrets/secretstore_update.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: external-secrets.io/v1beta1
+kind: SecretStore
+metadata:
+  name: secretstore-ibm
```

Comment:
Even this is will be one copy, can this be in the same namespace where you operator is..

Reply:
This yaml that updates SecretStore with ibm provider needs to be applied in all namespaces.
Having it in only the namespace where the operator exists, is not working.